unity/Assets/Maroon/reusableGui/Menu/PrefabsColumns/scrMenuColumnAudio.cs

```diff
@@ -20,18 +20,20 @@ public class scrMenuColumnAudio : MonoBehaviour
 
     void Start()
     {
-        // Listen to music slider
+        // Apply initial slider value from Music AudioSource and listen to music slider
         if (_musicSlider)
         {
-            _musicSlider.GetComponent<Slider>().onValueChanged
-                .AddListener((value) => this.OnChangeMusicSlider(value));
+            var musicSliderComponent = _musicSlider.GetComponent<Slider>();
+            musicSliderComponent.value = SoundManager.Instance.MusicVolume;
+            musicSliderComponent.onValueChanged.AddListener(this.OnChangeMusicSlider);
         }
-
-        // Listen to sound effect slider
+        
+        // Apply initial slider value from Sound Effect AudioSource and listen to sound effect slider
         if (_soundEffectSlider)
         {
-            _soundEffectSlider.GetComponent<Slider>().onValueChanged
-                .AddListener((value) => this.OnChangeSoundEffectSlider(value));
+            var soundEffectSliderComponent = _soundEffectSlider.GetComponent<Slider>();
+            soundEffectSliderComponent.value = SoundManager.Instance.SoundEffectVolume;
+            soundEffectSliderComponent.onValueChanged.AddListener(this.OnChangeSoundEffectSlider);
```

Comment:
SoundFX: Set initial slider value to its AudioSource's volume before adding the listener as before.